hil/ext/switches/dellnos9.py

```diff
@@ -338,9 +344,49 @@ def _is_port_on(self, port):
         assert shutdown in ('false', 'true'), "unexpected state of switchport"
         return shutdown == 'false'
 
+    def save_running_config(self):
+        command = 'write'
+        self._execute(EXEC, command)
+
+    def _get_config(self, config_type):
```

Comment:
So, this method is attached to a couple of other switch drivers, but isn't actually described in any of the interfaces, and appears to only be used by the test suite. and the leading underscore convention suggests that this is some internal-use method, so I was looking for uses elsewhere in the driver. This is kindof horrible; we shouldn't be breaking interface boundaries like this. Can we find a way to make it a bit more clear what this is here for, and that this is a public method for use by the test suite? I'm not super pleased by the way this is structured...

Reply:
yeah, breaking interface boundaries is bad. 

I just did what was approved in the past for the other drivers. ( #753, #749)

In any case, I'll address this and the other comment you made.